src/core/Akka/Event/Logging.cs

```diff
@@ -112,8 +112,15 @@ public static string FromActor(IActorContext actor, ActorSystem system)
 
         public static string FromActorRef(IActorRef a, ActorSystem system)
         {
-            var defaultAddress = system.AsInstanceOf<ExtendedActorSystem>().Provider.DefaultAddress;
-            return defaultAddress is null ? a.Path.ToString() : a.Path.ToStringWithAddress(defaultAddress);
+            try
+            {
+                var defaultAddress = system.AsInstanceOf<ExtendedActorSystem>().Provider.DefaultAddress;
+                return defaultAddress is null ? a.Path.ToString() : a.Path.ToStringWithAddress(defaultAddress);
+            }
+            catch // can fail if the ActorSystem (remoting) is not completely started yet
+            {
+                return a.Path.ToString();
+            }
```

Comment:
Second guard to make sure that, in the event that any exception is thrown, we still return a valid actor path